src/boilerplates/boilerplates.ts

```diff
@@ -93,6 +94,25 @@ export function makeCommandBoilerplates(program: Command) {
             createEmbeddedWalletNextjs(program, options);
         });
 
+    comm
+        .command("create-email-embedded-wallet-react-starter")
+        .description("Clone a starter boilerplate for Sequence Email authenticated Embedded Wallet and React")
+        .option(
+            "--waas-config-key <waas_key>",
+            "WaaS config key for this project"
+        )
+        .option(
+            "--project-access-key <access_key>",
+            "Project access key for Sequence requests"
+        )
+        .option(
+            "--verbose",
+            "Show additional information in the output"
+        )
+        .action((options) => {
+            createEmailEmbeddedWalletReact(program, options);
+        });
+
```

Comment:
Remove the indentation within each .option to maintain consistency with the other commands